Kickstarter-iOS/Features/PaginationExample/PaginationExampleView.swift

```diff
@@ -0,0 +1,82 @@
+import KsApi
+import SwiftUI
+
+private struct PaginationExampleProjectCell: View {
+  let title: String
+  var body: some View {
+    Text(title)
+      .padding(.all, 10)
+  }
+}
+
+private struct PaginationExampleProjectList: View {
+  @Binding var projectIdsAndTitles: [(Int, String)]
+  @Binding var showProgressView: Bool
+  @Binding var statusText: String
+
+  let onRefresh: @Sendable() -> Void
+  let onDidShowProgressView: @Sendable() -> Void
```

Comment:
@Sendable allows for these methods to run concurrently, yes?

Reply:
I initially put this tag here because Xcode insisted that I did (IIRC, something like these properties were implicitly `@Sendable` and I needed to make them explicit). I've never used `@Sendable` before, so I took a peek in the docs, and you are correct that it allows a function to be used "across concurrency domains." My guess is that here, pragmatically, it's enforcing something at compile time that's important - perhaps that every captured variable included in `onRefresh` and `onDidShowProgressView` won't be de-allocated? But we should probably learn more if this comes up frequently.

Reply:
Huh, y'know what? It compiles fine without it. Maybe this was an artifact of an earlier version of this code. I'm going to clean this up.